cloud_storage_handler/clients/minio.py

```diff
@@ -0,0 +1,22 @@
+"""Minio client module.
+
+This module provides functionality to interact with the Minio cloud storage
+service.
+"""
+
+from cloud_storage_handler.clients.minio_client import MinioClient
+
+
+def get_minio_client():
+    """Get a Minio client and create a bucket.
+
+    Returns:
+        MinioClient: An initialized Minio client with a created bucket.
+    """
+    init_client = MinioClient()
+    client = init_client.get_client()
+    init_client.create_bucket()
+    return client
+
+
+minio_client = get_minio_client()
```

Comment:
Not sure what you are trying to do here, but if this is a way of trying to persist a client instance, I'm pretty sure that won't work, as this module-level code is automatically executed everytime you import this module - meaning that you'd anyway get a new client instance everytime.

Besides, you probably don't always want to create a bucket, right?

If you want a new client instance, use `cloud_storage_handler.clients.minio_client.MinioClient().get_client()`. If you want to persist a client instance, pass it on from where you created it. For example you could use the `MinioConfig()` for that. Define an optional client instance param and then populate it. Like that, it will be available in app and available wherever you need it (if it's fine to use the same client instance for every request).

I guess this will become clearer once this is integrated with the endpoints, when you actually have the clients do some work.

For now, I would just remove the `return` statement from the function, remove this module-level instantiation of the function and rather call the init function directly in `cloud_storage_handler.main`.